libs/ha/ha_common_libs_k8s.py

```diff
@@ -1256,7 +1256,9 @@ def setup_mock_monitor(node_obj):
         return True
 
     def simulate_disk_cvg_failure(self, node_obj, source, resource_status, resource_type,
-                                  node_type='data', resource_cnt=1, node_cnt=1, delay=None):
```

Comment:
Can you please add datatypes in the parameters list and explain the same in doc string!

Reply:
Added datatypes in the parameters list and doc string as well